app/assets/javascripts/hyrax/editor/controlled_vocabulary.es6

```diff
@@ -114,9 +128,25 @@ export default class ControlledVocabulary extends FieldManager {
   // '_destroy' hidden parameter
   removeFromList( event ) {
       event.preventDefault()
+      // Changing behavior of the remove button to add a new field if the last field is removed
+      // Using querySelector to find elements with data-attribute="based_near"
+      const basedNearElements = document.querySelectorAll('[data-attribute="based_near"]');
+      const parentsArray = Array.from(basedNearElements).map(element => element.parentElement);
+      const nonHiddenElements = parentsArray.filter(element => element.style.display !== 'none');
+      const nonHiddenCount = nonHiddenElements.length;
+      if (nonHiddenCount < 2){
+        let $listing = $(event.target).closest(this.inputTypeClass).find(this.listClass)
+        let $activeField = $listing.children('li').last()
+        $listing.append(this.createNewField($activeField));
+        this.postRemovalAdjustment += 1;
```

Comment:
How is `postRemovalAdjustment` used? It seems like each time I delete the last based_near value, this goes up by one and means that the next input created will have a higher index. Were index collisions occurring without it when editing existing works?

Reply:
Yes that's correct. There was a bug causing index collisions after removing an isolated dropdown.